tests/testthat/test-as_model_out_tbl.R

```diff
@@ -39,57 +32,8 @@ test_that("column renaming works", {
   )
 })
 
-test_that("triming to task ids works", {
```

Comment:
For example, I've had to remove this entire test as the function requires a `hub_connection` object to determine task IDs.

I will revert back when `hubData` is on CRAN.